docs/modules/tiered-storage/pages/configuration.adoc

```diff
@@ -86,10 +86,10 @@ when the local disk is filled up, you will be able to flush data to, for example
 Tiered Storage works fine for most use cases with the default settings provided in <<configuring-tiered-storage, Configuring Tiered Storage>>.
 However, there may be specific use cases where you might want to change the behavior related to the device to be used.
 
-[tabs] 
+[tabs]
```

Comment:
@ethurma Do we want to make any commitment in the documentation about what we will support in the future? (section starting with line 80).

Reply:
No, we should avoid commenting on the roadmap and committing to features in the docs.